src/calendar/day/basic/index.tsx

```diff
@@ -75,13 +75,7 @@ export default class BasicDay extends Component<BasicDayProps> {
   };
 
   get marking() {
-    let marking = this.props.marking || {};
```

Comment:
It appears this code used to handle a case when marking is an array. 
Are we sure we want to remove it? 
If we're keeping it, there's definitely cleaner way to check for array

Reply:
We're not supporting markings as an Array...